README.md

```diff
@@ -257,15 +267,45 @@ Building the program with BLAS support may lead to some performance improvements
     cmake --build . --config Release
     ```
 
-### Prepare Data & Run
+### Aquiring Setting up the 7b model weights
+
+You can use this system to conduct research on an AI chatbot vaguely comparable to ChatGPT-3 and it will even run on your local machine without needing massive amounts of hardware.  But to do so you **must** install the Alpaca 7b model weights into the models folder.
+
+Because these resources belong to Facebook, their official path to obtaining the data should be followed.  While it's true that most researchers using the Alpaca weights obtained them from a magnet link to a torrent file, linking or sharing that magnet link should not be done in this repo due to the questionability of violating FaceBook's IP rights and also (not to be an alarmist here) the potential for the popularization of these weights to cause harm.
+
```

Comment:
Up until your edit, the 7B parameter models mentioned were the official LLaMA models from FaceBook. Suddenly this section starts talking about those 7B parameter models as if they were the Alpaca weights. This makes it sound like llama.cpp was designed for Alpaca. Perhaps you aren't familiar with the differences, but they are not the same thing. You may want to check your models against the [checksums provided](https://github.com/ggerganov/llama.cpp/blob/master/SHA256SUMS) to see if you downloaded the official models or some third party version of the model.

Quick aside, the Alpaca models do offer a somewhat ChatGPT like experience (with `-ins`), but it's weird to single them out at this point. There are others that work as well or better than the Alpaca models.

Also, by merging this "acquiring" section into the 7B example, it makes it sound like people should specifically acquire 7B-sized models when that was just the usage example. This works with all size LLaMA models.

And then why are we saying "most researchers" are using models they got from Torrents. I guess it's possible, but do we even know if that's true? I feel like if they were going to write a paper about it, they'd go through the proper channels to acquire the models.

I also don't think we need to warn people about harm. There are lots of free, open source models being released lately, this seems to unfairly point fingers at LLaMA.

Reply:
It is also not a requirement to install any models. 7B is not a requirement, why do you think so, @TheNotary?

Reply:
Oh, just one minor comment.  Regarding most researchers using a torrent, I don't think it's clear the extend to which Meta is making their content available publicly.  There are more reports of people filling out the form and then not hearing back from them so I believe on top of the massive popularity of the magnet link, I think it's a safe assumption to make even if it can't be verified with 100% certainty.